src/java/org/apache/cassandra/net/MessagingService.java

```diff
@@ -506,7 +506,7 @@ public void send(Message message, InetAddressAndPort to, ConnectionType specifyC
     {
         if (isShuttingDown)
         {
-            logger.error("Cannot send the message {} to {}, as messaging service is shutting down", message, to);
+            logger.warn("Cannot send the message {} to {}, as messaging service is shutting down", message, to);
```

Comment:
does this apply to earlier versions too ?

Reply:
Most likely to be a different cause in 5.0: the message causing the failure in CASSANDRA-19804 doesn't exist there, plus this wouldn't cause a timeout it's a hard failure.